Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Properties/ParameterDefinitionHandler.cs

```diff
@@ -5,13 +5,15 @@ namespace Speckle.Converters.RevitShared.ToSpeckle;
 /// </summary>
 public class ParameterDefinitionHandler
 {
-  private sealed record GroupDefinition(string Group, string? Units);
+  private sealed record ParameterDefinition(string GroupName, string? Units);
+
+  private sealed record ParameterKey(string InternalName, string Group);
 
   /// <summary>
   /// Keeps track of all parameter definitions used in the current send operation. This should be attached to the root commit object post conversion.
   /// </summary>
   /// POC: Note that we're abusing dictionaries in here because we've yet to have a simple way to serialize non-base derived classes (or structs?)
-  private readonly Dictionary<string, GroupDefinition> _groupDefinitions = new();
+  private readonly Dictionary<ParameterKey, ParameterDefinition> _parameterDefinitions = new();
```

Comment:
Because `record` types override `.Equals` and `.GetHasCode`, it is safe to use them as key right?

Reply:
yup!